.github/workflows/release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       matrix:
         helm: ['2.17.0', '3.4.2', '3.7.1']
     env:
-      FIXTURES_GIT_BRANCH: ${{ github.head_ref || github.ref_name }}
+      FIXTURES_GIT_BRANCH: ${{ github.base_ref || github.ref_name }}
```

Comment:
Can you please revert this change, I just merged https://github.com/aslafy-z/helm-git/pull/223 that should solve the issue.

Reply:
sure